infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -489,7 +489,7 @@ async function startServer() {
     const results: any = await enquirer.prompt(questions);
 
     let components: string[] = [];
-    const defaultChoices = ['http_api', 'eth', 'state_keeper', 'housekeeper', 'tree_lightweight'];
+    const defaultChoices = ['http_api', 'eth', 'state_keeper', 'housekeeper', 'tree'];
```

Comment:
Given that even EN currently requires full tree to run, the ability to use it in hyperchain setup should be reconsidered (e.g. I'd assume that it may only work in a microservice-based setup where both full and lightweight tree are ran independently). However given that the performance of full tree is pretty good anyways, probably it's not a very big deal for the time being.
cc @githubdoramon